### DISCUSSION TITLE
WebUI not accessable

### DISCUSSION THREAD

**Discussion by DhlDeliveryMan** [postalserver/postal#2012] [Help with using Postal] [answered]
Hello!

I installed postal as instructed, then i set up the nginx conf file for this domain, installed the ssl license added the reverse proxy etc.

Then when i go to the domain It keeps loading, and then **Unable to connect**.

All my other domains work, and the port `5000` is permit ed, so i doubt its a firewall issue.

my conf file:

```
server {
        listen 80;
        listen [::]:80;

        return 301 https://$host$request_uri;
        server_name mail.mydomain.com www.mail.mydomain.com;

        access_log off;
        error_log  /var/log/nginx/mail.app-error.log error;
}

server {
        listen [::]:443 ssl;
        listen 0.0.0.0:443 ssl;
        server_name mail.mydomain.com www.mail.mydomain.com;
        # root /opt/postal/public;
        ssl_certificate /etc/letsencrypt/live/mail.mydomain.com/fullchain.pem;
        ssl_certificate_key /etc/letsencrypt/live/mail.mydomain.com/privkey.pem;
        ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
        ssl_prefer_server_ciphers on;
        ssl_ciphers EECDH+ECDSA+AESGCM:EECDH+aRSA+AESGCM:EECDH+ECDSA+SHA512:EECDH+ECDSA+SHA384:EECDH+ECDSA+SHA256:ECDH+AESGCM:ECDH+AES256:DH+AESGCM:DH+AES256:RSA+AESGCM:!aNULL:!eNULL:!LOW:!RC4:!3DES:!MD>
        location / {
                client_max_body_size 50M;
                try_files $uri $uri/index.html $uri.html @puma;
        }
        location /app/assets {
        add_header Cache-Control max-age=3600;
}
location @puma {
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header Host $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto https;
        proxy_pass http://127.0.0.1:5000;
     }
}
```

One thing i did notice is when i cd into the `/opt/postal` directory,  the directory `/public` doesn't exist there. 
I think there may be a different way to configure the file, but i couldn't find it, so any help would be appreciated.

**Comment by willpower232** [comment, marked as answer]
> One thing i did notice is when i cd into the /opt/postal directory, the directory /public doesn't exist there.

The nginx config you have is for Postal v1 when it sounds like you're using v2.

You only need one `location` block

```
location / {
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header Host $host;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto https;
	proxy_pass http://127.0.0.1:5000;
}
```

**Reply by DhlDeliveryMan** [reply]
So I removed all of the `location` blocks and replaced it with the one you provided, however the issue persists.

Is there possibly something else i have to remove from my config for it to work ?

**Reply by willpower232** [reply]
> It keeps loading, and then Unable to connect

If this is your browser then this sounds like you need to open port 443 and or 80 on your firewall

**Reply by DhlDeliveryMan** [reply]
Ah. Yeah the good old issue. I did forget to open port 443. 

Thank you for the help.
